Ginger/GingerCoreCommon/Repository/BusinessFlowLib/BusinessFlow.cs

```diff
@@ -86,6 +86,20 @@ public enum eSource
             Selenium,   // From Selenium Import
             Gherkin     // From Gherking Feature file
         }
+        
+        bool mSelected;
```

Comment:
why still you keep the addition of "Selected" to BF if you using "BusinessFlowToConvert"? please remove it?

Reply:
Dont wanted to duplicate the same list so used this list of businessflow on the businessflow page and on the report page i used class "BusinessFlowToConvert", but now removed this property